core/node/base_token_fetcher/src/lib.rs

```diff
@@ -46,7 +51,7 @@ impl ConversionRateFetcher for NoOpConversionRateFetcher {
 #[derive(Debug)]
 pub struct BaseTokenFetcher {
     pub config: BaseTokenFetcherConfig,
-    pub latest_to_eth_conversion_rate: AtomicU64,
+    pub latest_to_eth_conversion_rate: Arc<StdMutex<BigDecimal>>,
     http_client: reqwest::Client,
     error_reporter: Arc<Mutex<ErrorReporter>>,
```

Comment:
Why do we have to combine both std and tokio mutex in the same structure? If we need `tokio` mutex just so that we can `.await` it (e.g. we do not hold the mutex guard across any await point), it should be safe to stick to `std` mutex everywhere (it's actually OK to use blocking mutexes even in asynchronous context, as stated even by [tokio's own docs](https://docs.rs/tokio/latest/tokio/sync/struct.Mutex.html#which-kind-of-mutex-should-you-use)).

Reply:
We have moved all tokio `mutex`es to std as suggested.